src/packageSourceGenerator/PackageSourceGeneratorTask/RewriteNuspec.cs

```diff
@@ -82,6 +84,9 @@ public override bool Execute()
             // Ensure that the nuspec's line endings match the current environment
             nuspecContent = nuspecContent.ReplaceLineEndings();
 
+            nuspecContent = nuspecContent.Replace("http://go.microsoft.com/fwlink/?LinkId=529443", MicrosoftMitLicenseUrl);
+            nuspecContent = nuspecContent.Replace("http://go.microsoft.com/fwlink/?LinkId=329770", MicrosoftMitLicenseUrl);
```

Comment:
My understanding of the agreement is that old FX packages that are Microsoft licensed can be updated to MIT.  @leecow to confirm.

Reply:
Do you mean "_can't_ be updated"?

Reply:
> Do you mean "_can't_ be updated"?

No - They can be updated to MIT.  That is what this change is doing.

Reply:
Ah, got it.

Reply:
I believe this is true but would need @richlander to confirm.

Reply:
I'm trying to grasp the scope of this. There are a lot of packages here. I looked at `Microsoft.Build.Framework`. It is MIT, so changing to MIT seems like a correction rather than needing an agreement. Are there some packages that are not MIT?

It seems like getting issues created for packages that are not MIT is the correct path. Yes?

Also, we should move off of `licenceUrl` to `licenseExpression`.

Reply:
LGTM

Reply:
Thanks for the detailed info.